ckanext/ontario_theme/templates/internal/package/snippets/related_datasets.html

```diff
@@ -17,7 +17,7 @@ <h2 class="ontario-h3">{{ _("Related datasets") }}</h2>
     <legend class="ontario-fieldset__legend">
       Are these suggestions accurate? <span class="ontario-label__flag">(required)</span>
     </legend>
-    <div class="ontario-radios">
+    <div class="ontario-radios" id="related-dataset-feedback-bottom">
       <div class="ontario-radios__item">
         <input class="ontario-radios__input"
                id="radio-button-option-yes"
```

Comment:
Can we change the value of this to "option-yes" and add the attribute `data-feedback="bottom"` to both radio buttons?
*see below for example